Entity/RedirectRoute.php

```diff
@@ -114,7 +119,29 @@ public function getSource()
      */
     public function setSource($source)
     {
-        $this->source = $source;
+        $this->source = '/' . ltrim($source, '/');
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSourceHost()
+    {
+        return $this->sourceHost;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setSourceHost($sourceHost)
+    {
+        if (!$sourceHost) {
```

Comment:
would prefer using `empty` here 🙂 
```suggestion
        if (!empty($sourceHost)) {
```